src/js/game/LevelMVC/LevelView.js

```diff
@@ -851,6 +851,9 @@ module.exports = class LevelView {
 
   playBlockSound(groundType) {
     var oreString = groundType.substring(0, 3);
+    if (groundType === "water" || groundType === "lava") {
+      return;
+    }
```

Comment:
Tiny nit: I'd rather see this above `var oreString = groundType.substring(0, 3);` so that line still has context for how it is used.